app/src/main/java/com/github/se/gomeet/ui/mainscreens/profile/Profile.kt

```diff
@@ -96,7 +97,7 @@ fun Profile(
       val allEvents =
           eventViewModel.getAllEvents()!!.filter { e -> currentUser!!.myEvents.contains(e.eventID) }
       allEvents.forEach {
-        if (!isPastEvent(it)) {
+        if (!isPastEvent(it) || isJoinedEvent(it, userId)) {
```

Comment:
This has been fixed in a previous PR, you should do a git pull origin main and take the new changes (so not these changes, the changes that had been pushed to the main branch)